docs/dev/news/5760.fix

```diff
@@ -0,0 +1 @@
+Enable revealing a function that's a member of a type, using a static receiver
```

Comment:
```suggestion
Enable revealing a function that's a member of a type, using a static receiver, in the new resolver
```
This was already working in the old resolver, correct?

Reply:
I think so. AFAIK I haven't touched the old resolver behavior, in this PR or others. Looking at an existing test for opaque and reveal, I see some static cases: https://github.com/dafny-lang/dafny/blob/master/Source/IntegrationTests/TestFiles/LitTests/LitTest/dafny0/OpaqueFunctions.dfy#L347

Reply:
In that case, may I suggest this:

```suggestion
The new resolver (accessible using `--type-system-refresh`) can now handle revealing instance functions using a static receiver, as it is the case for the current resolver
```

Otherwise I was confused in that someone reading the release notes might think "Well, this was already enabled in the last release" and won't understand that it's about the new resolver unless it's explicit. Not everyone is using the new resolver. which still fails on some codebases including the Dafny Core codebase itself.

Reply:
Ah, sorry I misread before and thought your suggestion was what I initially wrote. Yes this is much better, thanks.